app/src/main/AndroidManifest.xml

```diff
@@ -11,8 +11,8 @@
         android:name=".HumanLearningApplication"
         android:allowBackup="false"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
+        android:label="@string/app_name"
```

Comment:
that looks like an unnecessary diff, can you remove it?

Reply:
you are right. It's reverted now